content/java/developer-guide/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Developer Guide"
+date: 2017-07-13T22:52:55-04:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "java"
+projectname: "WorldWind Java"
+---
+
+## Developer Guide
```

Comment:
For this iteration, I'd like to avoid sub sections and place related list items next to one another. The site interface is usable with two levels of nesting (first level main and projects, second level project topics). We'd need additional UI work to make more levels usable, and I'd rather we just do what we can within the current layout constraints.

We could keep the concept of a Developer's Guide. However, the distinction between guides and tutorials is unclear to me. It seems to me that a straight port of goworldwind's structure has introduced a design inconsistency we don't want. How about sticking with Tutorials until we see a clear pattern for content we want to distinguish?